api_documentation_v1.yaml

```diff
@@ -0,0 +1,406 @@
+        actualLoadedAmountKg:
+          type: number
+          description: Actual amount loaded in mixer wagon (unit = KG)
+          format: double
+          example: 1980
+        amountPerLoadKg:
+          type: number
+          description: Amount that should be loaded in mixer wagon (unit = KG)
+          format: double
+          example: 2000
+        id:
+          type: integer
+          description: Feeding load ID
+          format: int64
+          example: 452
+        loadDetails:
+          description: Loading details
+          $ref: '#/components/schemas/LoadingDetailsExternalDTO'
+        loadPosition:
+          type: integer
+          description: Feeding load position used for ordering feeding loads
+          format: int64
+          example: 0
+        status:
+          type: integer
+          description: 'Feeding load status enum 0 = TO DO, 1 = IN PROGRESS , 2 = DONE '
+          format: int32
+          example: 0
+        unloads:
+          type: array
+          description: Unloads from feeding load per pen
+          items:
+            $ref: '#/components/schemas/UnloadExternalDTO'
+    FeedingLoadIngredientExternalDTO:
+      title: FeedingLoadIngredientExternalDTO
+      type: object
+      properties:
+        calculatedAmountKg:
+          type: number
+          description: Calculated amount of feed in feeding load (unit KG)
+          format: double
+          example: 780
+        feedId:
+          type: integer
+          description: Feed ID
+          format: int64
+          example: 254
+        feedName:
+          type: string
+          description: Feed name
+          example: Alfalfa Meal, 17% CP
+        id:
+          type: integer
+          description: Loading ID
+          format: int64
+        loadedAmountKg:
+          type: number
+          description: Actual amount of ingredient in feeding load (unit KG)
+          format: double
+          example: 750
+    FeedingReportExternalDTO:
+      title: FeedingReportExternalDTO
+      type: object
+      properties:
+        feedingLoads:
+          type: array
+          description: Feeding loads for given feeding
+          items:
+            $ref: '#/components/schemas/FeedingLoadExternalDTO'
+        feedingNumber:
+          type: integer
+          description: >-
+            Feeding number enum value 1 = MORNING FEEDING , 2 = NOON FEEDING, 3
+            = EVENING FEEDING
+          format: int64
+          example: 1
+        id:
+          type: integer
+          description: Feeding ID
+          format: int64
+          example: 256
+        pens:
+          type: array
+          description: Pens included in feeding
+          items:
+            $ref: '#/components/schemas/PenExternalDTO'
+        status:
+          type: integer
+          description: Feeding status enum value 0 = TO DO , 1 = IN PROGRESS , 2 = DONE
+          format: int32
+          example: 0
+    LoadingDetailsExternalDTO:
```

Comment:
I think that structure is currently too complicated for the atlas use-case. I could imagine a structure like the following:
````
[{ // one object per feeding-event
  ration_id: abc,
  loadings: [{
    startDate: 2023-01-01T13:10:00
    endDate: 2023-01-01T13:30:00
    ingredients: [{ 
      feedId: abc-def-hij,
      calculatedAmountKg: 780,
      loadedAmountKg: 750
    }],
    unloadings: [{
      startDate: 2023-01-01T14:00:00
      endDate: 2023-01-01T14:30:00,
      penId: 68,
      penNumberOfAnimals: 100
      penName: Test pen
      calculatedAmountKg: 910 // do you have this number in your recipe?
      unloadedAmountKg: 900
    }]
  }]
}]
````

Reply:
We do not have calculatedAmountKg, only unloadedAmountKg

Reply:
in the current version you have: unloadedAmountKg, is this measured or set by user as target?
I know some weights have both numbers, and this is very interesting for feedingreports. If you don't measure the output, I would still add it to this atlas-standard and make it optional.

Reply:
![grafik](https://user-images.githubusercontent.com/4425214/218963523-d6f65e80-6610-49e5-8570-2232ad95cade.png)